missas/core/management/commands/postgres2sqlite.py

```
@@ -0,0 +1,221 @@
+        """Copy all tables from source to target database."""
+        try:
+            with transaction.atomic(using=target_db):
+                self.clear_target_database(tables, target_db)
+
+                for table in tables:
+                    self.copy_table(table, source_db, target_db)
+
+        except Exception as e:
+            self.stdout.write(
+                self.style.ERROR(
+                    f"\n✗ Error during copy: {e}\n" "All changes have been rolled back."
```

Ruff (0.14.4)

Implicitly concatenated string literals on one line

(ISC001)